cypress/support/metaTestHelper.js

```diff
@@ -94,18 +114,25 @@ export const metadataAssertion = () => {
     const lastPublished = new Date(
       windowData.metadata.lastPublished,
     ).toISOString();
-
-    retrieveMetaDataContent('head meta[name="description"]', description);
-    retrieveMetaDataContent('head meta[name="og:title"]', title);
-    retrieveMetaDataContent('head meta[name="og:type"]', type);
-    retrieveMetaDataContent(
-      'head meta[name="article:published_time"]',
-      firstPublished,
-    );
-    retrieveMetaDataContent(
-      'head meta[name="article:modified_time"]',
-      lastPublished,
-    );
+    retrieveData(description, title, type, firstPublished, lastPublished);
     getElement('html').should('contain', language);
   });
 };
+
+export const retrieveAMPMetadata = data => {
```

Comment:
Im not sure why metadataAssertion and retrieveAMPMetadata are seperate, they seem to do almost exactly the same things? Maybe they can be consolidated?